src/main/kotlin/no/nav/faktureringskomponenten/controller/dto/FakturaserieRequestDto.kt

```diff
@@ -41,17 +44,17 @@ data class FakturaserieRequestDto(
     val referanseNAV: String,
 
     @field:Schema(
```

Comment:
Vi kan gjemme metadata `@field` og istedet bare lese `Schema(`  for å redusere kodestøy